### ISSUE TITLE
Change icon that appears on phone to reflect that it's MDS + Phonegap app

### ISSUE THREAD

**Issue by GoogleCodeExporter** [erhanselman/mobile-web-development-with-phonegap#16]
```
What steps will reproduce the problem?
1. create an app
2. load on phone
3. see that icon is one of the phonegap icons

What is the expected output? What do you see instead?
new icon indicating app is from MDS/Phonegap

What version of the product are you using? On what operating system?


Please provide any additional information below.


```

Original issue reported on code.google.com by `libby.ba...@gmail.com` on 18 Feb 2011 at 10:06

**Comment by GoogleCodeExporter** [comment]
```
1.2 will add this icon only for projects created with the bundled PhoneGap 
version.  Added as part of 
http://code.google.com/a/eclipselabs.org/p/mobile-web-development-with-phonegap/
source/detail?r=67
```

Original comment by `paul.beu...@gmail.com` on 21 Feb 2011 at 7:57
- Changed state: **Started**
- Added labels: **Milestone-Release1.2.0**, **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
Fixed with 1.2 release
```

Original comment by `paul.beu...@gmail.com` on 23 Feb 2011 at 4:24
- Changed state: **Fixed**
